main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        3) horn
-        4) 'phone control' controls
-    Name 2 things you can do with the:   Steering Wheel
-        1) adjust cruise control settings.
-        2) honk
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1) speedometer
-        2) fuel gauge
-        3) mileage
-        4) temperature gauge
-    Name 3 things you can do with the:   Instrument Cluster
-        1) check the current speed
-        2) check the current fuel level
-        3) check the total miles usage
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) air condition control
-        2) air circulation control
-        3) seat heating control
-    Name 3 things you can do with the:   Environment Controls
-        1) set indoor temperature
-        2) get outdoor air
-        3) heat the seat
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) music system
-        2) map system
-        3) phone system
-    Name 3 things you can do with the:   Infotainment System
-        1) listen to music
-        2) guide to destination
-        3) call
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) headrest
-        2) seat adjuster
-        3) seatbelt
-    Name 2 things you can do with the:   Seat
-        1) recline the seat
-        2) put on seatbelt
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) electric guitar
+5 properties:
+    1) volume knob
+    2) tone knob
+    3) pickup selection
+    4) string type
+    5) body wood
+3 things it can do:
+    1) adjust volume
+    2) change tone
+    3) give joy
+ */
+
+/*
+Thing 2) labtop computer
+5 properties:
+    1) cpu core
```

Comment:
These are objects.   You need to choose values.  like `cpu cache size` or `cpu clock speed`
- [ ] revise